grunttasks/l10n-htmllint.js

```diff
@@ -34,10 +34,10 @@ module.exports = function (grunt) {
         }
       }
       if (content !== '') {
-        grunt.file.write(path.join('<%= yeoman.tmp %>', pathname[1], pathname[2], pathname[3].replace('json', 'html')), content);
+        grunt.file.write(path.join('.tmp', pathname[1], pathname[2], pathname[3].replace('json', 'html')), content);
```

Comment:
We could potentially use something like this instead: `grunt.config('yeoman.tmp')`